flipperdevices/flipperzero-firmware
# [FL-2500] Change shadows direction and refactor status bar drawing code

# What's new

- Change shadows direction and refactor status bar drawing code

# Verification 

- Compile and upload
- Check status bar, shadows now on the right
- Check archive, shadows now on the right

# Checklist (For Reviewer)
